src/client/src/css/buttonStyles.module.css

```diff
@@ -1,14 +1,11 @@
 .buttonHighlightedBorder {
```

Comment:
Do we still need this rule? It is the same as `.buttonHighlighted` and it has no border (despite what name suggests)?

Reply:
good catch, it is the same thing so I replaced it and removed it